src/files.rs

```diff
@@ -136,6 +137,21 @@ impl Files {
     Ok(Response::new(Body::from(body.into_string())))
   }
 
+  fn download_icon(size: u8) -> PreEscaped<String> {
+    html! {
+    svg
+      width=(size)
+      height=(size)
+      fill="none"
+      stroke="black"
+      stroke-width=(2)
+      stroke-linecap="round"
+      stroke-linejoin="round" {
+        use xlink:href="/static/feather-sprite.svg#download" {}
```

Comment:
This is really cool, I had no idea that you could somehow use `#…` fragment specifiers to select an icon inside of an SVG file.

Reply:
Yeah, I agree.

I just found this deprecation notice: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href. So I think this should be just `href`.